examples/java/src/main/java/org/apache/beam/examples/MinimalWordCount.java

```diff
@@ -45,37 +46,29 @@
  * <pre>
  *   1. Reading data from text files
  *   2. Specifying 'inline' transforms
- *   3. Counting a PCollection
- *   4. Writing data to Cloud Storage as text files
+ *   3. Counting items in a PCollection
+ *   4. Writing data to text files
  * </pre>
  *
- * <p>To execute this pipeline, first edit the code to set your project ID, the temp
- * location, and the output location. The specified GCS bucket(s) must already exist.
+ * <p>To execute this pipeline, first edit the output file to a location on your local disk
+ * (look for the string 'YOUR_OUTPUT_FILE' near the bottom of the file).
+ * This example uses the {@link DirectRunner}, so writing to local disk is OK. When running
+ * on a distributed service, you would use an appropriate file service.
  *
  * <p>Then, run the pipeline as described in the README. It will be deployed and run with the
  * selected runner. No args are required to run the pipeline. You can see the results in your
- * output bucket in the GCS browser.
+ * output file.
  */
 public class MinimalWordCount {
 
   public static void main(String[] args) {
     // Create a PipelineOptions object. This object lets us set various execution
-    // options for our pipeline, such as the associated Cloud Platform project and the location
-    // in Google Cloud Storage to stage files.
+    // options for our pipeline, such as the runner you wish to use.
     PipelineOptions options = PipelineOptionsFactory.create();
 
-    // In order to run your pipeline, you need to make following runner specific changes:
-    //
-    // CHANGE 1/3: Select a Beam runner, such as DataflowRunner or FlinkRunner.
-    // CHANGE 2/3: Specify runner-required options.
-    // For DataflowRunner, set project and temp location as follows:
-    //   DataflowPipelineOptions dataflowOptions = options.as(DataflowPipelineOptions.class);
-    //   dataflowOptions.setRunner(DataflowRunner.class);
-    //   dataflowOptions.setProject("SET_YOUR_PROJECT_ID_HERE");
-    //   dataflowOptions.setTempLocation("gs://SET_YOUR_BUCKET_NAME_HERE/AND_TEMP_DIRECTORY");
-    // For FlinkRunner, set the runner as follows. See {@code FlinkPipelineOptions}
-    // for more details.
-    //   options.setRunner(FlinkRunner.class);
+    // For this example, we hardcode the DirectRunner. In subsequent examples we will demonstrate
```

Comment:
To keep this example as simple as possible, we hardcode the DirectRunner. Later examples will show how to make this parametrizable.